.github/workflows/node.js.yml

```diff
@@ -12,12 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - name: Use Node.js 16.x
+    - name: Download and use Node.js 16.x
       uses: actions/setup-node@v3
       with:
         node-version: 16.x
-        cache: 'npm'
-    - run: npm ci
-    - run: npm run build --if-present
-    - name: Run tests
-      run: npm test
+    - run: yarn install
+    - name: Test
+      env:
+        ACCESS_TOKEN: ${{secrets.ACCESS_TOKEN}}
+      run: yarn test
```

Comment:
why you are changing npm by yarn?

Reply:
I was reading about yarn and it seemed cool to implement it in the project and it works the same.

Reply:
Ok 👍